src/commonControls/multiValueListControl/MultiValueListControlAPL.ts

```diff
@@ -380,3 +382,471 @@ export namespace MultiValueListControlAPLPropsBuiltIns {
         };
     }
 }
+
+export type MultiValueListStyles = 'checkBoxes' | 'dualList' | 'aggregateDuplicates';
+
+export namespace MultiValueListControlComponentAPLBuiltIns {
+    export function renderComponent(
+        control: MultiValueListControl,
+        props: MultiValueListAPLComponentProps,
+        input: ControlInput,
+        resultBuilder: ControlResponseBuilder,
+    ) {
+        assert(props.valueRenderer !== undefined);
+        if (props.renderStyle === 'checkBoxes') {
```

Comment:
pls breakout as separate functions and perhaps as separate files.

Reply:
split them into individual functions